AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -92,13 +90,7 @@ - (void)applyEnabledState:(BOOL)isEnabled {
       [self configAuthenticationClient];
       eTag = [MS_USER_DEFAULTS objectForKey:kMSIdentityETagKey];
     }
-    NSString *authToken = [self retrieveAuthToken];
-    NSString *accountId = [self retrieveAccountId];
-
-    // Only set the auth token if auth token and account id are not nil to avoid triggering callbacks.
-    if (authToken && accountId) {
-      [[MSAuthTokenContext sharedInstance] setAuthToken:authToken withAccountId:accountId];
-    }
+    [[MSAuthTokenContext sharedInstance] cacheAuthToken];
```

Comment:
Is it loading history from cache? Please take a look at Device and Session history. It can be loaded at the time AuthTokenContext is initialized.

Reply:
It can't be loaded without `Identity` module

Reply:
We probably need to think about this one because I'm not sure if it can be loaded without Identity module or we don't actually care. Session and Device histories are also loaded by core and these are not used by Distribute and Push modules. I don't see any reasons of coupling AuthTokenContext with Identity because it can be used without Identity in the future.